.github/workflows/cicd.yml

```diff
@@ -0,0 +1,83 @@
+name: ci
```

Comment:
Can you also rename the workflow name to cicd?

Reply:
done